documentation/modules/cruise-control/con-rebalance-goals-proposals.adoc

```diff
@@ -0,0 +1,180 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='con-rebalance-goals-proposals-{context}']
+= Rebalance goals and proposals
+
+To rebalance a Kafka cluster, Cruise Control uses _rebalance goals_ to generate _rebalance proposals_. 
+
+[discrete]
+== Rebalance goals
+
+Rebalance goals specify targets for resource distribution and utilization across a Kafka cluster. 
+They are organized in a customizable hierarchy, from high priority goals to low priority goals. 
+You can designate certain rebalance goals as _hard goals_ that must be satisfied--for more information, see link:#soft-hard-goals[Soft goals and hard goals].
+
+With a few exceptions, cluster rebalancing in {ProductName} supports the same goals as the Kafka Cruise Control project (where the term "optimization goals" is used). 
```

Comment:
What does it mean "...supports the same goals as the Kafka Cruise Control"? Strimzi is using Kafka Cruise Control! Thinking on this, if the CC community names them "optimization goals" why we should change to "rebalance goals"? Isn't it better having consistency?

Reply:
I was under the impression that Strimzi will not support _all_ of the CC optimization goals. Is that not the case?

Re. "rebalance goals" vs "optimization goals", the integration with Cruise Control is for cluster rebalancing, not the other CC features such as Anomaly Detection, Topic Replication Factor. Therefore, we felt that "rebalance goals" was a better name.

But I'm happy to rename to "optimization goals" -  @tomncooper, do you have any strong opinions?

Reply:
Yes, optimization goals is a better name, not least because eventually we may well support most of the CC features.

Reply:
Good point. Time to Find and Replace... ;)